pulpcore/cli/common/generic.py

```diff
@@ -331,74 +331,60 @@ def _version_callback(
     return value
 
 
-# TODO: would be great to have enable this to take a validator, rather than having
-# to build "on top of" it like I'm doing now w/ json_callback
-def load_file_or_string_callback(
-    ctx: click.Context, param: click.Parameter, value: Optional[str]
+def load_file_wrapper(
+    handler: Callable[[click.Context, click.Parameter, str], Any] = lambda c, p, x: x
```

Comment:
I think, i'd be better to not have the default value here, but pass the lambda in line 364.
`load_string_callback = load_file_wrapper(lambda c, p, x: x)`